dali/pipeline/util/thread_pool.cc

```diff
@@ -60,20 +60,19 @@ ThreadPool::~ThreadPool() {
 }
 
 void ThreadPool::AddWork(Work work, int64_t priority, bool start_immediately) {
-  bool started_before = false;
+  bool should_start_all = false;
```

Comment:
I think we should make this a separate PR, for better traceability.

Reply:
Yeah, I guess.